frontend/src/metabase/dashboard/containers/DashboardApp.unit.spec.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Route } from "react-router";
+import { screen } from "@testing-library/react";
+
+import userEvent from "@testing-library/user-event";
+import fetchMock from "fetch-mock";
+import { renderWithProviders, waitForElementToBeRemoved } from "__support__/ui";
+import DashboardApp from "metabase/dashboard/containers/DashboardApp";
+import {
+  createMockCollection,
+  createMockDashboard,
+  createMockUser,
+} from "metabase-types/api/mocks";
+import {
+  setupCollectionItemsEndpoint,
+  setupCollectionsEndpoints,
+  setupDashboardEndpoints,
+} from "__support__/server-mocks";
+import { createMockDashboardState } from "metabase-types/store/mocks";
+import { BEFORE_UNLOAD_UNSAVED_MESSAGE } from "metabase/hooks/use-before-unload";
```

Comment:
1. You can import everything you can find in `@testing-library/react` from `__support__/ui`; could we please keep only one import?
2. Not sure why linter doesn't complaint, but `metabase/hooks` import should come _before_ `metabase-types` imports

Reply:
Hmm yeah the linter doesn't seem to care where I put the `metabase/hooks` import as long as it isn't with the 3rd party library imports. Not sure what happened there, I checked my eslint config and it's pointing to our `eslintrc` file